builder/js/ncf.js

```diff
@@ -1672,16 +1672,16 @@ $scope.onImportFileChange = function (fileEl) {
   }
 
   $scope.getResourcesInfo = function(){
-    var msg, nbResources, nbUncommitted;
+    var msg, nbResources, nbUnsaved;
     nbResources = ($scope.selectedTechnique !== undefined && $scope.selectedTechnique.resources !== undefined) ? $scope.selectedTechnique.resources.length : undefined;
     if(nbResources === undefined || nbResources <= 0){
       msg = "There is no resource files."
     }else{
-      nbUncommitted = $scope.selectedTechnique.resources.filter(resource => resource.state != 'untouched').length;
-      if(nbUncommitted <= 0){
+      nbUnsaved = $scope.selectedTechnique.resources.filter(resource => resource.state != 'untouched').length;
+      if(nbUnsaved <= 0){
         msg = "There " + (nbResources   > 1 ? "are " : "is ") + nbResources   + " resource file" + (nbResources > 1 ? "s." : ".");
```

Comment:
Funny thing i learned, but to think in the future, 0 is plural in english, and not singular, so it should be nbResources === 1 ? is : are